scripts/jenkins/jobs-obs/openstack-trackupstream.yaml

```diff
@@ -129,7 +133,10 @@
               "openstack-nova-virt-zvm",
               "openstack-ec2-api",
               "python-networking-cisco",
-              "python-networking-hyperv" ].contains(component) )
+              "python-networking-hyperv" ].contains(component) ||
+            [ "Cloud:OpenStack:Liberty:Staging", "Cloud:OpenStack:Kilo:Staging", "Cloud:OpenStack:Juno:Staging" ].contains(project) &&
+            [ "openstack-magnum",
+              "openstack-horizon-plugin-magnum-ui" ].contains(component) )
```

Comment:
can you add those to the previous "project" sections in the lines above instead of adding a new one?